requirements.txt

```diff
@@ -0,0 +1,8 @@
+-e git+http://github.com/openaustralia/scraperwiki-python.git@morph_defaults#egg=scraperwiki
```

Comment:
Okay, sounds great. I had some issues working with the morph.io API since they use JSONP, but it sounds doable with a tiny bit of rewriting. For the time being, I'll follow this up with a PR to add a *scraperwiki-requirements.txt*.